test/Mono.Linker.Tests.Cases/DataFlow/GetTypeDataFlow.cs

```diff
@@ -69,6 +71,22 @@ static void TestConstructors ()
 			type.RequiresNone ();
 		}
 
+		[ExpectedWarning ("IL2072", nameof (DataFlowTypeExtensions.RequiresAll) + "(Type)", nameof (Type.GetType) + "(String)")]
+		static void TestNull ()
+		{
+			// Warns about the return value of GetType, even though this throws at runtime.
+			Type.GetType (null).RequiresAll ();
```

Comment:
Can you please file an issue about this? I don't think it's super important to fix, but we should track it as it's inconsistent right now, some intrinsics handle this without warnings, and some don't.